postgre/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM openjdk:8-jre-alpine
+
+# Create an avatica user
+RUN addgroup -S avatica && adduser -S -G avatica avatica
+RUN mkdir -p /home/avatica/classpath
+
+ARG AVATICA_VERSION
+ARG POSTGRESQL_VERSION="42.0.0"
```

Comment:
Should this be parameterised?  Or just use the latest (since it's not really going to matter much)?  And isn't it being set by the shell script above?